textsum/seq2seq_attention.py

```diff
@@ -46,10 +46,10 @@
 tf.app.flags.DEFINE_string('mode', 'train', 'train/eval/decode mode')
 tf.app.flags.DEFINE_integer('max_run_steps', 10000000,
                             'Maximum number of run steps.')
-tf.app.flags.DEFINE_integer('max_article_sentences', 2,
+tf.app.flags.DEFINE_integer('max_article_sentences', 100,
```

Comment:
I don't think it can handle 100 sentences. 2~4 sentences and ~100 words is where it can handle.